lib/y2common.pm

```diff
@@ -0,0 +1,168 @@
+package y2common;
```

Comment:
How about just snapper? It's not really common for more yast modules than just snapper, right?